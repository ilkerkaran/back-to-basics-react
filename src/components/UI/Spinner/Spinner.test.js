import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Spinner from './Spinner';


describe('<Spinner /> snapshot ', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Spinner/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('<Spinner /> ', () => {
  it(' renders div witha a class name Loading', () => {
    const spinner = shallow(<Spinner/>);
    expect(spinner.text()).toEqual('Loading...');
  });
});
