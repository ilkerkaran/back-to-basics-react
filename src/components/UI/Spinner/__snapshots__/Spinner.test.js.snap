// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> snapshot  renders correctly 1`] = `
<div
  className="Loader"
>
  Loading...
</div>
`;
